Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8.1-slim
+
+ENV PYTHONPATH=/app
+
+RUN mkdir /app
+COPY requirements.txt /app/requirements.txt
+
+RUN pip install -r /app/requirements.txt
+
+# copy everything else after pip install to take advantage of docker image layers (cache)
+COPY . /app
+
+WORKDIR /app
```

Comment:
Adding `CMD` + `EXPOSE` would be nice for documentation and useful if anybody is running this locally.

Suggestion:
```dockerfile
EXPOSE 5000
CMD ["gunicorn", "awslimits.server:app", "--bind", "0.0.0.0:5000"]
```